assets/l10n/app_es.arb

```
@@ -0,0 +1,474 @@
+  "tagCocktail": "Cóctel",
+  "tagTropicalCocktail": "Cóctel Tropical",
+  "tagMateTea": "Té de Mate",
+  "tagMilk": "Leche",
+  "tagKamaboko": "Kamaboko",
+  "tagOden": "Oden",
+  "tagCheese": "Queso",
+  "tagEgg": "Huevo",
+  "tagFriedEgg": "Huevo Frito",
+  "tagButter": "Mantequilla",
+  "done": "Hecho",
+  "cancel": "Cancelar",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)